src/core/annotation.js

```diff
@@ -3433,6 +3435,10 @@ class TextAnnotation extends MarkupAnnotation {
 
     super(params);
 
+    // No rotation for Text (see 12.5.6.4).
+    this.noRotate = true;
```

Comment:
I'm guessing that you actually meant the following here?
```suggestion
    this.data.noRotate = true;
```